### PR TITLE
loading issue and speed optimization

### PR DESCRIPTION
**Pull request by dev-rajneeshkumar**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.live/products/microplate-readers/multi-mode-readers/spectramax-i3x-readers
- After: https://moldev-opt--moleculardevices--hlxsites.hlx.live/products/microplate-readers/multi-mode-readers/spectramax-i3x-readers

### CODE REVIEW

File: blocks/header/helpers.js [hunk 1]
```diff
@@ -50,7 +51,7 @@ export function expandMenu(element) {
   element.setAttribute('aria-expanded', 'true');
 }
 
-export function buildBrandLogo(content) {
+export async function buildBrandLogo(content) {
```
Comment (mhaack) [inline blocks/header/helpers.js:54]: This is not doing much I wonder how making it async and `await buildBrandLogo(content)` should improve any performance?

File: blocks/latest-resources/latest-resources.js [hunk 2]
```diff
@@ -29,7 +30,6 @@ async function getResourcesFromMetaTags() {
 
   return ffetch('/query-index.json')
     .sheet('resources')
-    .chunks(2000)
```
Comment (duynguyen) [inline blocks/latest-resources/latest-resources.js:32]: @dev-rajneeshkumar just to make sure you are aware of this change: with 2k chunk size if will only take 1 call to fetch all resources to run the filter (at this point of time with 1.9k resources). with default chunk size (255) it takes 8 calls for the same number of resources. And in the end, the filter will run through all 1.9k resources.

Comment (mhaack) [reply blocks/latest-resources/latest-resources.js:32]: Plus we habe HTTP/2 should not matter mucht.

**Review by mhaack — APPROVED** [review]

File: blocks/latest-resources/latest-resources.js (hunk 2 above)
Comment (dev-rajneeshkumar) [reply blocks/latest-resources/latest-resources.js:32]: ok @duynguyen let see the performance and user experience after this PR merge and then will change if required

**Pull request merged**